lxc/image.go

```diff
@@ -388,29 +388,13 @@ func (c *imageCmd) run(conf *config.Config, args []string) error {
 			return err
 		}
 
-		// Attempt to resolve an image alias
-		var imgInfo *api.Image
-		image := inName
-		if c.copyAliases {
-			alias, _, err := d.GetImageAlias(image)
-			if err == nil {
-				image = alias.Target
-			}
-
-			// Get the image info
-			imgInfo, _, err = d.GetImage(image)
-			if err != nil {
-				return err
-			}
-		} else {
-			// Don't fetch full image info if we don't need aliases (since it's
-			// an expensive operation)
-			imgInfo = &api.Image{}
-			imgInfo.Fingerprint = image
-			imgInfo.Public = true
+		image := c.dereferenceAlias(d, inName)
+		imgInfo, _, err := d.GetImage(image)
+		if err != nil {
+			return err
```

Comment:
And now simplestreams will take forever again...

I'll send another branch that takes care of all the cases.